app/dao/bookings.js

```diff
@@ -4,6 +4,8 @@ const { query, save, update } = require('../services/dbService.js');
 
 const { BOOKINGS_TABLE } = require('../../config/all.js');
 
+const { parseCurrentDate } = require('../services/dateService.js');
```

Comment:
You can move that service import above constant import, next to the rest of services imports.